client/src/styles.scss

```diff
@@ -265,6 +286,63 @@ button.mat-menu-item.selected {
     font-weight: bold !important;
 }
 
+.meta-text-block .mat-icon-button {
```

Comment:
Should probably be in the .scss of `meta-text-block-component`

Reply:
it does not work in meta-text-block-component.scss.